compiler/Cargo.toml

```diff
@@ -7,16 +7,16 @@ license = "Apache-2.0"
 readme = "README.md"
 
 [dependencies]
-ink_primitives = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_metadata = { git = "https://github.com/paritytech/ink", default-features = false, features = ["derive"], optional = true }
-ink_env = { git = "https://github.com/paritytech/ink", default-features = false, features = ["ink-debug"] }
-ink_storage = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_lang = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_prelude = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_eth_compatibility = { git = "https://github.com/paritytech/ink", default-features = false }
+ink_primitives = { version = "3.0.0-rc9", default-features = false }
```

Comment:
The `version` in the manifest is still set to `rc8`.

Again, I'd try and pull this from the `cargo-contract` provided template to ensure that it's actually up to date.

Reply:
The problem is that the Carg.toml neeeds to include **all** ink! dependencies, not just the ones provided from the `Cargo.toml` which is provided by `cargo-contract`.